tests/monitor/test_workload_perf_mon.py

```diff
@@ -48,20 +56,33 @@ def test_single_sample(self):
     def test_monitor_cpu_usage_normalization(self):
```

Comment:
@rostyboost I have not substantially modified the behavior of the actual normalization code.  It looked to me like this test had incorrect assumptions based on two problems
1. Timestamps were in backwards order
1. The initial max timestamp was based on the _current_ time instead of the last timestamp from the input.

Can you have a look to check?